pkg/event/iterator.go

```diff
@@ -69,7 +70,19 @@ func (i *Iterator) Next(ctx context.Context) (event Event, err error) {
 			return event, ErrOutOfSync
 		}
 	}
-	return
+	return event, err
+}
+
+// NextForFamilyTable blocks and returns the next event that matches the specified family and table
+func (i *Iterator) NextForFamilyTable(ctx context.Context, family, table string) (event Event, err error) {
```

Comment:
so if int-consumer wanted to use this, it would be to open an iterator for every table it wanted to track?

Reply:
Yes, do you see any issues with that?

Reply:
Correct, the use of `Next` and `NextForFamilyTable` together would not make sense and would not be the intended usage model. Perhaps to make this clearer a separate struct would be better. Will modify and update shortly.

Reply:
looks great, thank you for doing that